client/scripts/views/components/proposal_card.ts

```diff
@@ -253,11 +253,11 @@ const ProposalCard: m.Component<{ proposal: AnyProposal, injectedContent? }> = {
         // thread link
         proposal.threadId && m('.proposal-thread-link', [
           m('a', {
-            href: `/${app.activeId()}/proposal/discussion/${proposal.threadId}`,
+            href: `/${app.activeChainId()}/proposal/discussion/${proposal.threadId}`,
```

Comment:
Are we also deprecating `app.activeId()` now that there's only `chain`?

Reply:
Yup. Although whether we use `activeId()` vs `activeChainId()` is up to devs' preference. I like the latter as it's more explicit.